service-node/backend/src/test/scala/org/enterprisedlt/fabric/service/node/UtilTest.scala

```diff
@@ -85,4 +87,31 @@ class UtilTest extends FunSuite {
     }
 
 
+    test("fillPlaceholdersEnvironmentVariables should work fine") {
+
+        val targetProperty = Array(
+            Property("a", "1"),
+            Property("b", "2"),
+            Property("c", "3")
+        )
+
+        val values = Array(
+            Property("a", "1"),
+            Property("b", "2"),
+            Property("c", "3")
+        )
+
+        val dictionary = Array(
+            Property("a", "1"),
+            Property("b", s"$${b}"),
+            Property("c", s"$${c}")
+        )
+
+        val filledProperty = Util.fillPlaceholdersProperties(values, dictionary)
+
+        filledProperty shouldEqual  targetProperty
```

Comment:
probably has to be `sameElements` for correct arrays comparison

Reply:
done.